bin/can_server.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+import logging
+FORMAT = '%(asctime)-15s %(message)s'
+logging.basicConfig(format=FORMAT)
+import argparse
+import can
+from can.interfaces import remote
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description="Remote CAN server")
+
+    parser.add_argument('-v', action='count', dest="verbosity",
+                        help='''How much information do you want to see at the command line?
+                        You can add several of these e.g., -vv is DEBUG''', default=2)
+
+    parser.add_argument('-c', '--channel', help='''Most backend interfaces require some sort of channel.
```

Comment:
I guess that would make sense. My thought was that since we do not have any official default bitrate today, then if the application does not specifiy a bitrate it will be backend dependent. However, since the application and physical CAN bus is separated here, the application might not know what the bitrate should be, therefore not providing one and relies on the server to know what bitrate is appropriate.